signbank/settings/development.py

```diff
@@ -11,7 +11,7 @@
 #: Debug should be True in development but not in production!
 DEBUG = os.environ.get('DEBUG', 'false').lower() == 'true'
 
-DEFAULT_ALLOWED_HOSTS = ['127.0.0.1', 'localhost', 'signbank.nz']
+DEFAULT_ALLOWED_HOSTS = ['127.0.0.1', '0.0.0.0', 'localhost', 'signbank.nz']
```

Comment:
Can you give more background to this change?

Reply:
Mistake! I should not have checked devlopment.py in. On my machine the host defaults to 0.0.0.0 for some reason.
I'll add a backout commit for this file.

Reply:
Fixed.